bitnami/keycloak/Chart.yaml

```diff
@@ -26,4 +26,4 @@ name: keycloak
 sources:
   - https://github.com/bitnami/bitnami-docker-keycloak
   - https://github.com/keycloak/keycloak
-version: 6.2.3
```

Comment:
Please remove this change from the PR since it belongs to keycloak

Reply:
Done.Please review it again.Thank you^_^